logback-core/src/main/java/ch/qos/logback/core/BasicStatusManager.java

```diff
@@ -90,18 +97,18 @@ private void fireStatusAddEvent(Status status) {
 
     public void clear() {
         synchronized (statusListLock) {
-            count = 0;
+            count.reset();
```

Comment:
Should level be reset as well?

```suggestion
            count.reset();
            level.set(Status.INFO);
```